sensor_msgs/CompressedImage input_image
---
sensor_msgs/CompressedImage image
int16 num_detections
string[] classes
float32[] scores
sensor_msgs/RegionOfInterest[] bounding_boxes
---
